charts/sftd/templates/statefulset.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "sftd.fullname" . }}
+  labels:
+    {{- include "sftd.labels" . | nindent 4 }}
+spec:
+  # TODO: Make configurable in follow-up PR
+  # This is 1 on purpose as we need more machinery to make multiple SFTs work.
+  # Work for that is tracked in: https://github.com/wireapp/wire-server-deploy/pull/383
+  replicas: 1
+  serviceName: {{ include "sftd.fullname" . }}
+  selector:
+    matchLabels:
+      {{- include "sftd.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+    {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      labels:
+        {{- include "sftd.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      terminationGracePeriodSeconds: {{ .Values.terminationGracePeriodSeconds }}
+      hostNetwork: true
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: POD_IP
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.podIP
+          args:
+            - "-I" # Address for HTTP requests
+            - "$(POD_IP)"
+            - "-M" # Address for metrics requests
+            - "$(POD_IP)"
+            - "-A" 
+            - "{{ required "Must specify mediaAddress" .Values.mediaAddress  }}"
```

Comment:
TODO: this won't work. The pod might be rescheduled and then end up on another node. This needs to be dynamic